### ISSUE TITLE
Signoff all code.

### ISSUE THREAD

**Issue by karlnicholas** [CS111C-META-FALL-2016/LabB-Team1#6]

**Comment by karlnicholas** [comment]
EntryWayListInterface and StackOfBooks all look good to me. There is some implication in the text that StackOfBooks should be based on "Titles", which I assume means Strings, but since the assignment said to do what we wanted I'm happy with using Generics like we did.

**Comment by alternatebow** [comment]
Same here. I was a bit wary of the whole 'title' constraint, but generics should fulfill that requirement.

**Comment by jillHuang7** [comment]
I think the generic does that.
I have question here. What's 'signoff' mean here? logout code??

**Comment by karlnicholas** [comment]
Hi Jill, 
"signoff", or sign off, or sign-off is like when you sign a letter that you approve of. See [sign off](http://www.merriam-webster.com/dictionary/sign%20off) for definition. Here it means to put a comment that you are okay with the project being submitted to insight the way it is.

Looks like we're all happy with it, closing the code, moving to last issue.

**Comment by jillHuang7** [comment]
Thank you so much Karl for the nice, and clear description! I got it!

**Comment by gunterlugo** [comment]
Hey there! I joined Data Structures really late in the semester and I am just about caught up with all you guys. (I got an extension for all the assignments from Prof. Masters up until Week 3). 

I am pretty new to GitHub and I was wondering if anyone has some time to walk me through it this week? I'd love to start contributing and helping out with all of our HW assignments and labs. Thanks and let me know. I am free all week, happy to meet in person or remotely!
